app.js

```diff
@@ -1,11 +1,24 @@
 const express = require("express")
 const  { 
-    getApi
+    getApi,
+    getTopics
 } = require('./controllers/api.controller')
 
 const app = express()
 app.use(express.json())
 
 app.get('/api', getApi)
+app.get('/api/topics', getTopics)
+
+app.use((err, req, res, next) => {
+ if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg })
+ } else {
+    next(err)
```

Comment:
Are we using this error handling at the moment? If not, it can be removed until needed